_data/devices/spes.yml

```diff
@@ -0,0 +1,69 @@
+architecture: arm64
+battery:
+  capacity: 5000
+  removable: false
+  tech: Li-Po
+bluetooth:
+  profiles:
+  - A2DP
+  spec: '5'
+cameras:
+- flash: Dual LED (dual tone)
+  info: 50 MP (wide) + 8 MP (ultrawide) + 2 MP (macro) + 2 MP (depth)
+- front flash: None

```

Comment:
I think you have to remove "front" from here.. May be that's the reason for validation failed

Reply:
Yes, I had time to check this just today, I will test the validation now